frontend/public/components/deployment.tsx

```diff
@@ -91,7 +92,18 @@ const DeploymentDetails: React.FC<DeploymentDetailsProps> = ({obj: deployment})
     <div className="co-m-pane__body">
       <SectionHeading text="Deployment Overview" />
       {deployment.spec.paused && <WorkloadPausedAlert obj={deployment} model={DeploymentModel} />}
-      <DeploymentPodCounts resource={deployment} resourceKind={DeploymentModel} />
+      <TopologyDataController
```

Comment:
The topology data controller fetches too much information to be used like this for rendering the pod rings.
We should fetch only what's needed, if the data isn't already available.

Reply:
Added ```PodRingDataController``` https://github.com/openshift/console/pull/2091/commits/e6e5ca5404c64bc4d63b35265223979bc261f0c0